src/components/newsfeed/NewsItem.js

```diff
@@ -184,13 +184,13 @@ const NewsItem = (props) => {
               {
                 props.newsObject.relationships.authorizer.data.id &&
                 props.newsObject.relationships.authorizer.data.type &&
-                authorizer && 
+                authorizer &&
                 authorizer.attributes.name ?
                 <Grid item>
                   <Chip
-                    className={props.authorizerNameFilter === authorizer.attributes.name ? classes.activeFilterChip : null}
-                    onClick={() => props.toggleAuthorizerNameFilter(authorizer.attributes.name)}
-                    label={ 
+                    className={props.authorizerNameFilter?.attributes?.name === authorizer.attributes.name ? classes.activeFilterChip : null}
```

Comment:
This is the fix for the chip, the filter could be null. Previously this check was in the boolean check above, which is why it would filter out the location and authorizer chips. If you added an authorizer/scope filter, the chip would display properly